src/main/java/com/qaprosoft/carina/demo/mobile_components/swaglabapp/Services.java

```diff
@@ -0,0 +1,7 @@
+package com.qaprosoft.carina.demo.mobile_components.swaglabapp;
+
+import com.qaprosoft.carina.demo.mobile_components.swaglabapp.utils.AutoLogInUtil;
+
+public class Services {
```

Comment:
Instead of `Services` create `AbstractSwagLabTest` which extends `AbstractTest`. This class should be intermediate layer between `AbstractTest` and `SwagLabsTest`. All common code for your tests can be moved there